### ISSUE TITLE
Belt brush points wrong way

### ISSUE THREAD

**Issue by sparr** [Nexela/PickerExtended#86]
When I use belt brush, it always reorients to point north, instead of the direction I was already holding/using the belt on my cursor.

**Comment by JohnyDL** [comment]
Yeah this is a limitation of the game interface and been asked for before but cannot be fixed with the game how it is at the moment (there's just no way for a mod to know the orientation of a held item) https://github.com/Nexela/PickerExtended/issues/35

**Comment by Nexela** [comment]
I think there is a roundabout way to get this information. I will look into
it and let you know.
